### PR TITLE
[+] Command Generate - Prevent error when table name is a JS reserved word (#3upt3a)

### PR DESCRIPTION
**Pull request by guillaumedeslandes**

### CODE REVIEW

File: utils/strings.js [hunk 1]
```diff
@@ -1,5 +1,24 @@
 const _ = require('lodash');
 
+const reservedWords = [
```
Comment (arnaud-moncel) [inline utils/strings.js]: I think we can add `module` no ?
what happened if 
```js
const { module } = require('model');

module.exports = { };
```

Comment (guillaumedeslandes) [reply utils/strings.js]: will do

Comment (larcin) [inline utils/strings.js]: [Code Style] I think that we tend to have hard coded const like that capitalized and underscored:
```suggestion
const RESERVED_WORDS = [
```

File: utils/strings.js [hunk 2]
```diff
@@ -9,10 +28,18 @@ module.exports = {
...
 
+  isReservedWord(input) {
+    return reservedWords.includes(_.toLower(input));
+  },
+
   transformToSafeString(input) {
     if (/^[\d]/g.exec(input)) {
       return `model${input}`;
     }
+    // Notice: add dash to get proper snake/pascal case
```
Comment (larcin) [inline utils/strings.js]:
```suggestion
    // NOTICE: add dash to get proper snake/pascal case
```

File: utils/strings.js [hunk 3]
```diff
@@ -9,10 +28,18 @@ module.exports = {
     return _.camelCase(input);
   },
 
+  isReservedWord(input) {
+    return reservedWords.includes(_.toLower(input));
```
Comment (larcin) [inline utils/strings.js]:
```suggestion
    return RESERVED_WORDS.includes(_.toLower(input));
```

**Pull request merged**